pkgs/development/libraries/qt-5/modules/qtbase.nix

```
@@ -29,6 +29,9 @@
 
 let
   debugSymbols = debug || developerBuild;
+  qtPlatform = plat: with plat;
+    if isAarch64 && isLinux then "linux-arm-generic-g++"
+    else throw "please add a qtPlatform entry for ${plat.config}";
```

```suggestion
    else throw "Please add a qtPlatformCross entry for ${plat.config}";
```
where can we find the special cross string?